packages/world/src/utils/PlayerUtils.sol

```diff
@@ -63,6 +63,27 @@ function requireInPlayerInfluence(Vec3 playerCoord, EntityId entityId) view retu
   return coord;
 }
 
+// Checks if the player is in range of the fragment (8x8x8 cube)
+function requireFragmentInPlayerInfluence(Vec3 playerCoord, EntityId fragmentEntityId) view returns (Vec3) {
```

Comment:
this is neat, what I did in the past is make the client pass in the force field coord, but this lets us skip that